### PR TITLE
Add Dagobert secrets

### PR DESCRIPTION
**Pull request by corvus-ch**

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc [hunk 1]
```diff
@@ -173,6 +173,25 @@ vault kv put clusters/kv/${TENANT_ID}/${CLUSTER_ID}/global-backup \
...
+
+  *)
+    echo "Can not determine copy source for the Dagobert secrets" >&2
+    export DAGOBERT_SOURCE=""
+  ;;
+esac
+
+# Copy the Dagobert OpenShift Node Collector Credentials
+vault kv get -format=json "${DAGOBERT_SOURCE}" | jq '.data.data' \
+  | vault kv put -cas=0 "clusters/kv/${TENANT_ID}/${CLUSTER_ID}/dagobert" -
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc:194]: Maybe better if we only try to copy dagobert credentials if we actually found a source

```suggestion
if [ -n "${DAGOBERT_SOURCE}" ]; then
vault kv get -format=json "${DAGOBERT_SOURCE}" | jq '.data.data' \
  | vault kv put -cas=0 "clusters/kv/${TENANT_ID}/${CLUSTER_ID}/dagobert" -
fi
```

**Review by simu — APPROVED** [review]

File: docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc (hunk 1 above)
Comment (corvus-ch) [reply docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc]: I plaed it nice and added even more output.

**Review by simu — APPROVED** [review]

**Comment by tobru** [comment]
As this is not cloudscale.ch specific, I'd say this belongs into a partial which can be reused for other installation documentation. And as this is really an ugly hack, copying from any random OCP4 cluster, I'd either point to the password manager, store the secret on another more generic location to copy from or at least make a note to SYN-356.

**Comment by corvus-ch** [comment]
I disussed this with @simu and we agreed to do the follwoing:

* Place the Dagobert copy source in `clusters/kv/template/dagobert`
* Mention this workaround in SYN-356.

We also considered to go with `shared` or `global` instead of `template`. We opted for `template` as it is less likely to colide with what might come out of SYN-356.

For those not having access to SYN-356: It is a story in VSHNs internal ticket system and it is about the ability to have secrets defined on the global and the tenant level.

**Review by simu — APPROVED** [review]

**Comment by srueg** [comment]
We can add a Vault policy for clusters to access `clusters/kv/data/global/*`: https://git.vshn.net/syn/synfra-apps/-/merge_requests/44

**Review by tobru — CHANGES_REQUESTED** [review]
Please correct the typo in the commit message

**Review by tobru — APPROVED** [review]

**Pull request merged**